hs_file_types/models/geofeature.py

```diff
@@ -157,7 +157,7 @@ def get_xml(self, pretty_print=True):
             self.originalcoverage.add_to_xml_container(container_to_add_to)
 
         return CoreMetaData.XML_HEADER + '\n' + etree.tostring(RDF_ROOT, encoding='UTF-8',
-                                                               pretty_print=pretty_print)
+                                                               pretty_print=pretty_print).decode()
```

Comment:
This code is a little weird, in the sense that it first encodes to utf-8 and THEN re-decodes to Unicode, before -- in subsequent code -- html-encoding the result into a file!